src/commands/fun/Horoscope.js

```diff
@@ -0,0 +1,77 @@
+const { Command } = require("discord.js-commando");
+const fetch = require("node-fetch");
+const signs = [
+    "capricorn",
+    "aquarius",
+    "pisces",
+    "aries",
+    "taurus",
+    "gemini",
+    "cancer",
+    "leo",
+    "virgo",
+    "libra",
+    "scorpio",
+    "sagittarius"
+  ];
+
+function UserException(sign){
+    this.sign=sign;
+    this.name='UserException'
+}
+
+function isSign(sign){
+    if(signs.filter(e=> sign === e)[0] !== undefined){
+        return sign;
+    } else {
+        throw new UserException('Invalid sign, use: ')
+    }
+}
+
+module.exports = class Horoscope extends Command {
+  constructor(client) {
+    super(client, {
+      name: "horoscope",
+      group: "fun",
+      memberName: "horoscope",
+      description: "Gets a daily horoscope for your astrological sign.",
+      examples: ["horoscope carpircorn"],
+      throttling: {
+        usages: 1,
+        duration: 10
+      },
+      args: [
+        {
+          key: "sign",
+          prompt: "What is your sign?",
+          type: "string"
+        }
+      ]
+    });
+  }
+  async run(msg, { sign }) {
+    let day = "tomorrow";
+    let outlook = '';
+
+    try {
+        isSign(sign);
+        outlook = await fetch(
+            `${process.env.HOROSCOPE}?sign=${sign}&day=${day}`,

```

Comment:
I say bring in the horoscope scraping into the bot command if possible.

Reply:
Also the more we stick in the dotenv the more steps are needed during setup so maybe avoid that. Not much of an issue for our single server but if we're planning for this bot to be used beyond the server it could be annoying for users.